oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandler.java

```diff
@@ -542,26 +539,56 @@ private Map<String, String> buildDimensions(
         return dimensions;
     }
 
+    private Optional<APIGatewayProxyResponseEvent> validateRequest(
```

Comment:
To be honest I only moved this to a method because sonar didn't like the introduction of a nested try-catch (despite this already being present). A bit of a mouthful but I've renamed to `generateAuthenticationErrorResponseIfRequestInvalid`

Reply:
Makes sense - thanks for that